### DISCUSSION TITLE
Alcinoe.MongoDB.Client multiple documents.

### DISCUSSION THREAD

**Discussion by ViaCriativa** [MagicFoundation/Alcinoe#321] [General]
In Alcinoe.MongoDB.Client, how can I search (using aMongoDBClient.SelectData) and work with a result that brings more than one document?
For example, in the MongoDB result:
/* 1 */
{    "_id" : ObjectId("65ddf108d9b99bf334561358"),
    "name" : "TEST_01"
}
/* 2 */
{
    "_id" : ObjectId("65ddf123d9b99bf33456135c"),
    "name" : "TEST_02"
}
/* 3 */
{
    "_id" : ObjectId("65ddf129d9b99bf334561364"),
    "name" : "TEST_03"
}
How can I get the search result in this format? 
And how can I work with this result in Alcinoe.JSONDoc?

**Comment by Zeus64** [comment]
you will receive the result in Json format inside a TALJsonNodeA, so you only need to play with the json (IE: 

MyResultJson.childnodes[0].childnodes['name'].text
or if you are not sure 'name' exist in all node :
MyResultJson.childnodes[0].GetChildNodeValueText('name', '');

**Reply by ViaCriativa** [reply]
Thanks.
